src/lib/seam/connect/routes/workspaces.ts

```diff
@@ -0,0 +1,36 @@
+// TODO: This file is generated from route spec
+import type { Routes } from '@seamapi/types/connect'
+import type { Axios } from 'axios'
+import type { SetNonNullable } from 'type-fest'
+
+export class Workspaces {
+  client: Axios
+
+  constructor(client: Axios) {
+    this.client = client
+  }
+
+  async get(params: WorkspacesGetParams = {}): Promise<Workspace> {
+    const {
+      data: { workspace },
```

Comment:
Generally seems good, slight worry around destructuring but i think it's generally safe

Reply:
Should be ok, but certainly can just do this instead. I'm actually more concerned that the generated types are nullable. I guess the API can return an error object, but from the sdk pov anything that it not the expected response will throw.

```
   const {
      data,
    } = await this.client.get<WorkspacesGetResponse>('/workspaces/get', {
      params,
    })
    return data.workspace
```